_posts/2017-11-30-foreman-community-newsletter---november-2017.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: Foreman Community Newsletter - November 2017
+date: 2017-11-30 12:59:00
+author: Ori Rabin
+tags:
+- foreman
+- newsletter
+- community
+excerpt: |
+  Migrating from mailing lists, Foreman 1.16, Katello 3.5 RC, and the usual round-up
+---
+
+Important news for mailing list users! Also releases, demos, and all
+that goodness!
+
+### Migrating from Google Groups
+
+Last month Greg made a proposal to migrate away from the Google Groups
+mailing list to a dedicated Foreman community forum. You can read more
+about his reasoning [here][discourse_users], or watch the
+[AMA](https://www.youtube.com/watch?v=IdV6US_d4-U).
+
+The debate is not closed yet, so please do join in, either on the list
+thread above, or on the [migration poll][migration_poll] if you have views
+on this. At this time, the likely outcome is that we *will* move to
+Discourse in the near future, so watch out for more announcements about
+that in the coming weeks.
+
+### Foreman 1.16 released
+
+Foreman 1.16 was released today! This version took longer than usual
+but it was worth the wait. There are a lot of improvements including
+adding puppet 5 support, VMware changes, role locking for plugin and more.
+Go check out the full list on the [announce list][1_16]. As always bugs
+can be reported on [issue tracker][issues] and we appreciate getting reports
+and of course fixes!
+
+### Katello 3.5 RC2
+
+Katello 3.5 released RC2 on Nov 9th, see the [post][k_3_5_rc2] for details.
+The final version will be released very soon so watch out for announcements!
+As always, we'd love your feedback, so do report
+any issues you find to the [Katello bug tracker][k_issues]!
+
+### Community lead
+
+Greg, our community lead, will be taking time off for the next 5 months.
+He's still around on irc and the mailing lists but not as active as usual.
+Until the beginning of May please feel free to contact Ori Rabin,
+orabin on irc.
+
+### New Sponsors?
+
+Fastly, our new sponsor has provided us with a [cdn] for the website.
```

Comment:
Was `[cdn]` meant to be a link?

Reply:
This is actually a link. If the names match this is sufficient. So you could rename k_issues at the bottom to ```Katello bug tracker``` and simplify that link if you want.

Reply:
It was but I can change it.